daemon/config/config_test.go

```diff
@@ -546,7 +554,9 @@ func TestReloadWithDuplicateLabels(t *testing.T) {
 	flags := pflag.NewFlagSet("test", pflag.ContinueOnError)
 	flags.String("config-file", configFile, "")
 	flags.StringSlice("labels", lbls, "")
-	err := Reload(configFile, flags, func(c *Config) {})
+	err := Reload(configFile, flags, func(c *Config) {
+		assert.Check(t, is.DeepEqual(c.Labels, []string{"foo=the-same"}))
```

Comment:
Do we need to validate somehow that this assertion actually fired?  (I guess all the "reloaded when we expected it to" tests above do that?)

Reply:
Good point; let me add the `reloaded` bool for this one as well